### DISCUSSION TITLE
helpwave impulse

### DISCUSSION THREAD

**Discussion by felixevers** [helpwave/helpwave#45] [Hackathon]
Introducing `helpwave impulse` (working name) the smart way of fixing healthcare issue of entire cities.
We will take a look at the major healthcare problems the average of people in a city has - measured against experience and statistics from local clinics.
In the next step we will introduce and recommend actions to the all citizens (mobile app user) containing some healthy recipes, 10-minute back workouts for desk workers, stress management or relaxation exercises. All these actions will lead to better health and vitality over the entire city.
Users of the app get these measures directly to their phones in the most immersive way possible and are encouraged by their neighborhood group dynamics to do something good for themselves and the health of the city.

**Comment by aegis301** [comment]
Cool idea, let's include some gamification into this. City quarters against each other, streets, individuals. We could initiate actions like a bonus programm with coupons etc. for individuals reaching a certain value of points. Or big events where the street with the most points will get benefits...

**Comment by DasProffi** [comment]
Maybe we also want to extend our scope to not just be medical issues like health, but also focus on civic issues and something like outside of work events to improve the city ("Spontanes Ehrenamtliches Engagement" would be my German translation of it). Many people would like to do something like that, but don't want to be bound to fixed dates/schedules, so what if we create a platform to facilitate this.
Create an event like a yoga course in the park and make it accessible on the platform.
Maybe also things like picking up trash in a certain region of the city to clean it up (such events exist already).
And if we want to look in the future, maybe extend this by the local city government to post things that people want to be done and are approved by the city, but lack funding. Now people could assign themselves to do such tasks and help directly to improve their city. Like git hub just translated to city projects.
I am unsure whether something like this exists or whether it's just so highly bureaucratized, that it's too difficult.
But if implemented in an accessible way, it could bring together citizens, improve the city and create a sense of belonging to the city.
